app/controllers/myaccount/SuccessfulPasswordRecoverController.java

```diff
@@ -0,0 +1,58 @@
+package controllers.myaccount;
+
+import com.commercetools.sunrise.framework.components.controllers.PageHeaderControllerComponentSupplier;
+import com.commercetools.sunrise.framework.components.controllers.RegisteredComponents;
+import com.commercetools.sunrise.framework.controllers.SunriseContentController;
+import com.commercetools.sunrise.framework.controllers.WithQueryFlow;
+import com.commercetools.sunrise.framework.controllers.cache.NoCache;
+import com.commercetools.sunrise.framework.controllers.metrics.LogMetrics;
+import com.commercetools.sunrise.framework.hooks.EnableHooks;
+import com.commercetools.sunrise.framework.reverserouters.SunriseRoute;
+import com.commercetools.sunrise.framework.reverserouters.myaccount.recoverpassword.RecoverPasswordReverseRouter;
+import com.commercetools.sunrise.framework.template.TemplateControllerComponentsSupplier;
+import com.commercetools.sunrise.framework.template.engine.ContentRenderer;
+import com.commercetools.sunrise.framework.viewmodels.content.PageContent;
+import com.commercetools.sunrise.myaccount.authentication.recoverpassword.recover.viewmodels.SuccessfulRecoveryPageContentFactory;
+import play.mvc.Result;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletionStage;
+
+@LogMetrics
+@NoCache
+@RegisteredComponents({
+        TemplateControllerComponentsSupplier.class,
+        PageHeaderControllerComponentSupplier.class
+})
+public final class SuccessfulPasswordRecoverController extends SunriseContentController implements WithQueryFlow<Void> {
```

Comment:
I just realized that you used a different Controller. How is it? I thought you were going to just add a "sucessful" action to the `RecoverPasswordController` and modify the template so that it would show a successful message, optionally hiding the form.